tachyon/zk/plonk/vanishing/circuit_polynomial_builder.h

```diff
@@ -99,8 +99,8 @@ class CircuitPolynomialBuilder {
     value_parts.reserve(num_parts_);
     // Calculate the quotient polynomial for each part
     for (size_t i = 0; i < num_parts_; ++i) {
-      VLOG(1) << "BuildExtendedCircuitColumn part: (" << i << " / "
-              << num_parts_ - 1 << ")";
```

Comment:
(Optional) I guess this change(78bcc2a) would be better to describe as a `refac` instead of `feat`.

Reply:
This also makes sense!